src/Goerli-DssSpell.t.base.sol

```diff
@@ -635,7 +635,7 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
                 assertEq(uint256(clip.buf()), normalizedTestBuf, concat("TestError/clip-buf-", ilk));
                 assertTrue(clip.buf() >= RAY && clip.buf() <= 2 * RAY, concat("TestError/clip-buf-range-", ilk)); // gte 0% and lte 100%
                 assertEq(uint256(clip.tail()), values.collaterals[ilk].clip_tail, concat("TestError/clip-tail-", ilk));
-                if (ilk == "TUSD-A") { // long tail liquidation
+                if (ilk == "TUSD-A" || ilk == "GUSD-A" || ilk == "USDC-A" || ilk == "PAXUSD-A") { // long tail liquidation
```

Comment:
consider to add a more granular if-block range check for the stables in the spell, as `tail` is just 12 hours, maybe along with the comment:
`// medium tail liquidation`

Reply:
I just extended the default one slightly.